docs/customizing.md

```diff
@@ -245,3 +245,275 @@ Example `pulp_install_plugins` (with bogus version values):
+  vars:
+    pulp_redis_bind: 'example-redis-server:6379'
+  roles:
+    - pulp_redis
+
+- hosts: example-pulp-api-server
+  vars:
+    pulp_default_admin_password: << YOUR PASSWORD FOR THE PULP APPLICATION HERE >>
+    pulp_api_bind: example-pulp-api-server:24817
+    pulp_settings:
+      secret_key: << YOUR SECRET HERE >>
+      content_origin: "http://example-webserver.fqdn"
+      redis_host: example-redis-server
+      redis_port: 6379
+      databases:
+        default:
+          HOST: example-postgres-server
+          ENGINE: django.db.backends.postgresql_psycopg2
+          NAME: pulp
+          USER: pulp
+          PASSWORD: << YOUR DATABASE PASSWORD HERE >>
+    pulp_install_plugins:
+      pulp-container: {}
+      pulp-rpm: {}
+  roles:
+    - pulp_api
+    - pulp_database_config
+  environment:
+    DJANGO_SETTINGS_MODULE: pulpcore.app.settings
+
+- hosts: example-pulp-content-server
+  vars:
+    pulp_content_bind: example-pulp-content-server:24816
+    pulp_settings:
+      secret_key: << YOUR SECRET HERE >>
+      content_origin: "http://example-webserver.fqdn"
+      redis_host: example-redis-server
+      redis_port: 6379
+      databases:
+        default:
+          HOST: example-postgres-server
+          ENGINE: django.db.backends.postgresql_psycopg2
+          NAME: pulp
+          USER: pulp
+          PASSWORD: << YOUR DATABASE PASSWORD HERE >>
+    pulp_install_plugins:
+      pulp-container: {}
+      pulp-rpm: {}
+ roles:
+   - pulp-content
+ environment:
+   DJANGO_SETTINGS_MODULE: pulpcore.app.settings
+
+- hosts:
+   - example-pulp-worker-server1
+   - example-pulp-worker-server2
+ vars:
+   pulp_settings:
+     secret_key: << YOUR SECRET HERE >>
+     content_origin: "http://example-webserver.fqdn"
+     redis_host: example-redis-server
+     redis_port: 6379
+     databases:
+       default:
+         HOST: example-postgres-server
+         ENGINE: django.db.backends.postgresql_psycopg2
+         NAME: pulp
+         USER: pulp
+         PASSWORD: << YOUR DATABASE PASSWORD HERE >>
+   pulp_install_plugins:
+     pulp-container: {}
+     pulp-rpm: {}
+ roles:
+   - pulp_workers
+ environment:
+   DJANGO_SETTINGS_MODULE: pulpcore.app.settings
+
+- hosts: example-resource-manager-server
+ vars:
+   pulp_settings:
+     secret_key: << YOUR SECRET HERE >>
+     content_origin: "http://example-webserver.fqdn"
+     redis_host: example-redis-server
+     redis_port: 6379
+     databases:
+       default:
+         HOST: example-postgres-server
+         ENGINE: django.db.backends.postgresql_psycopg2
+         NAME: pulp
+         USER: pulp
+         PASSWORD: << YOUR DATABASE PASSWORD HERE >>
+   pulp_install_plugins:
+     pulp-container: {}
+     pulp-rpm: {}
+ roles:
+   - pulp_resource_manager
+ environment:
+   DJANGO_SETTINGS_MODULE: pulpcore.app.settings
+
+- hosts: example-webserver
+ vars:
+   pulp_content_bind: example-pulp-server:24816
+   pulp_api_bind: example-pulp-server:24817
```

Comment:
This is copied and pasted from the prior example, but does not apply to this one.

The hostnames differ.

Reply:
would you mind telling me explicitly what needs to change? In your blog, they seem to be also copy pasted so I've nothing to judge against.

Reply:
I've added "suggestion" comments with the changes.